spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/service/impl/TaskServiceUtilsTests.java

```diff
@@ -183,6 +183,22 @@ public void testExtractAppProperties() {
 		assertThat(result.get("test")).isEqualTo("baz");
 	}
 
+	@Test
+	public void testExtractAppLabelProperties() {
+		Map<String, String> taskDeploymentProperties = new HashMap<>();
+		taskDeploymentProperties.put("app.mylabel.foo", "bar");
+		taskDeploymentProperties.put("test.none", "boo");
```

Comment:
**[TinyNit]** Having the `name = test` and the property name `test` is confusing. I think using something like:
```
name = myappName
label = myappLabel
prop name = myProp
```
would make it clear.